intranet/apps/polls/models.py

```diff
@@ -94,6 +94,23 @@ def get_users_voted(self):
                 users = list(q.get_users_voted())
         return users
 
+    def get_num_eligible_voters(self):
+        users = []
+        if self.groups.all():
+            for g in self.groups.all():
+                if users:
+                    users = list(set(DjangoGroup.objects.get(name=g).user_set.all()) | set(users))
+                else:
+                    users = list(DjangoGroup.objects.get(name=g).user_set.all())
+            return len(users)
+        else:
+            users = get_user_model().objects.all()
```

Comment:
Use `return get_user_model().objects.count()  - 1` to avoid loading all users into memory.